api/WGSLLanguageFeatures.json

```diff
@@ -79,6 +79,44 @@
           }
         }
       },
+      "extension_unrestricted_pointer_parameters": {
+        "__compat": {
+          "description": "<code>unrestricted_pointer_parameters</code> extension",
+          "spec_url": "https://gpuweb.github.io/gpuweb/wgsl/#language_extension-unrestricted_pointer_parameters",
+          "tags": [
+            "web-features:webgpu"
+          ],
+          "support": {
+            "chrome": {
+              "version_added": "123"
```

Comment:
Why is there no note for this?

Reply:
Again, it really should be there. Added in all places where it is missing.